readme.md

```diff
@@ -7,10 +7,10 @@ In order to run this sample you will need a number Environment variables configu
 | Key Name | Purpose |
 |----------|---------|
 | APIMEVENTS-EVENTHUB-NAME  | Azure Event hub name configured to receive events from API Management service|
-| APIMEVENTS-EVENTHUB-CONNECTIONSTRING |  Azure Event hub configuration string |
+| APIMEVENTS-EVENTHUB-CONNECTIONSTRING | Azure Event hub configuration string eg: `Endpoint=sb://<sb-url>/;SharedAccessKeyName=<name>;SharedAccessKey=<key>` |
 | APIMEVENTS-STORAGEACCOUNT-NAME | Azure Storage Account used for keeping track of what events have been read |
 | APIMEVENTS-STORAGEACCOUNT-KEY | Key for Azure Storage Account|
-| APIMEVENTS-MOESIF-APP-ID | Application Id for Moesif |  
+| APIMEVENTS-MOESIF-APP-ID | Moesif Collector Application Id (from portal "api keys") |  
```

Comment:
nitpic: can we use similar description as we do for kong-plugin-moesif and other sdks. I think thats very descriptive of moesif applicationId

```config.application_id: Your Moesif Application Id can be found in the Moesif Portal. After signing up for a Moesif account, your Moesif Application Id will be displayed during the onboarding steps.
You can always find your Moesif Application Id at any time by logging into the Moesif Portal, click on the top right menu, and then clicking Installation.

```
https://github.com/Moesif/kong-plugin-moesif/blob/master/README.md